boards/arm/atsamd21_xpro/atsamd21_xpro.dts

```diff
@@ -132,3 +208,36 @@ zephyr_udc0: &usb0 {
 	pinctrl-0 = <&usb_dc_default>;
 	pinctrl-names = "default";
 };
+
+/* The below Xplained Pro Extension Header labels are provided to
+ * assist shield setup. They do not reconfigure the SERCOM modules,
+ * select the correct driver, or adjust pin muxes. These changes
+ * must be made by overlays at the shield and application levels.
+ */
+
+ext1_spi: &sercom0 {
+};
+
+ext1_i2c: &sercom2 {
+};
+
+ext1_serial: &sercom4 {
+};
+
+ext2_spi: &sercom1 {
+};
+
+ext2_i2c: &sercom2 {
+};
+
+ext2_serial: &sercom4 {
+};
+
+ext3_spi: &sercom5 {
+};
+
+ext3_i2c: &sercom2 {
+};
+
+ext3_serial: &sercom4 {
+};
```

Comment:
If pins are not assigned to these, how can they be routed to a specific shield function?